README.md

```diff
@@ -270,6 +270,30 @@ type FactoryProps = {
 };
 ```
 
+## Troubleshooting
+
+`react-stripe-elements` may not work properly when being used with components that implement `shouldComponentUpdate`. `react-stripe-elements` relies heavily on React's `context` feature and `shouldComponentUpdate` does not provide a way to take context updates into account when deciding whether to allow a re-render and hence can block context updates from reaching components.
```

Comment:
nit: run-on -> "...deciding whether to allow a re-render. These components can block context updates from reaching `react-stripe-element` components in the tree."